articles/migrations/0002_auto_20240220_1730.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+from django.core.management import call_command
+
+
+def load_initial_data(apps, schema_editor):
+    call_command(
+        "loaddata",
+        "article_status.json",
+        "request_status.json",
+        app_label="articles",
+    )
+
+def unload_data(apps, schema_editor):
+    call_command("flush", verbosity=0, interactive=False)
```

Comment:
This **flush** command deletes not only the data in **ArticleStatus** or **ReviewRequestStatus** tables. But all the data in entire database. What you should do here is remove the data in a selected table